i18n/index.js

```diff
@@ -2,6 +2,9 @@ import get from 'lodash/get'
 import { lookup } from 'country-data-lookup'
 import createDebug from 'debug'
 import isRenderer from 'is-electron-renderer'
+import PropTypes from 'prop-types'
+
+export const intlShape = PropTypes.instanceOf(Object)
```

Comment:
Why is this defined as a simple `Object` type here? Shouldn't it be `IntlShape` as per the seemingly unused `i18n/intjShape.js` file?

Reply:
resolved